src/app/games/game/game-progress/game-progress.component.scss

```diff
@@ -9,27 +9,49 @@ hr {
   float: left;
 }
 
-.value {
+// .value.metric {
+//   min-width: 4em;
+//   text-align: right;
+// }
+
+.matching {
+  color: #01ce47;
+}
+
+.nonmatching {
+  color: #ffc107;
+}
```

Comment:
As above, could a more appropriate name other than "matching/nonmatching" be used here, as they are not necessarily these?

Reply:
Resolving for the same reason as above.